api/api.py

```diff
@@ -35,7 +35,7 @@
     'uid': '[0-9a-zA-Z.@_-]*',
 
     # Container name
-    'cname': 'projects|sessions|acquisitions|collections',
+    'cname': 'projects|sessions|acquisitions|collections|analyses',
```

Comment:
I've added this change as issue https://github.com/scitran/core/issues/856  for later because it did not seem to be non-trivial